script/surface-connections.lua

```diff
@@ -0,0 +1,60 @@
+function ClearAllSurfaceConnections()
+  global.ConnectedSurfaces = {}
+end
+
+local function sorted_pair(number1, number2)
+  return (number1 < number2) and (number1..'|'..number2) or (number2..'|'..number1)
+end
+
+local function lazy_subtable(a_table, key)
```

Comment:
use flib_table.get_or_insert

Reply:
The current [flib docs](https://factoriolib.github.io/flib/modules/table.html) don't list it as an available function (anymore?) but it's still there. I'm reluctant to use it, though, because creating a table for the default_value that won't be used in almost all cases seems wasteful.

Reply:
FLIB doc system broke back in 2021 when it was either that or autocomplete for vscode.
It was recently moved to sourcehut to get automated documentation running again.